hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/BlockNodeSimulatorOp.java

```diff
@@ -0,0 +1,559 @@
+                    "Block node simulator operations require BlockNodeMode.SIMULATOR to be set. " + "Current mode: "
+                            + network.getBlockNodeMode() + ". "
+                            + "Set system property 'hapi.spec.blocknode.mode=SIM' to enable simulator mode.");
+        }
+
+        BlockNodeSimulatorController controller = network.getBlockNodeSimulatorController();
+        long verifiedBlock = 0;
+
+        switch (action) {
+            case SEND_END_OF_STREAM_IMMEDIATELY:
+                verifiedBlock = controller.sendEndOfStreamImmediately(nodeIndex, responseCode, blockNumber);
+                log.info(
+                        "Sent immediate EndOfStream response with code {} for block {} on simulator {}, last verified block: {}",
+                        responseCode,
+                        blockNumber,
+                        nodeIndex,
+                        verifiedBlock);
+                break;
+            case SEND_SKIP_BLOCK_IMMEDIATELY:
+                controller.sendSkipBlockImmediately(nodeIndex, blockNumber);
+                verifiedBlock = controller.getLastVerifiedBlockNumber(nodeIndex);
+                log.info(
+                        "Sent immediate SkipBlock response for block {} on simulator {}, last verified block: {}",
+                        blockNumber,
+                        nodeIndex,
+                        verifiedBlock);
+                break;
+            case SEND_RESEND_BLOCK_IMMEDIATELY:
+                controller.sendResendBlockImmediately(nodeIndex, blockNumber);
+                verifiedBlock = controller.getLastVerifiedBlockNumber(nodeIndex);
+                log.info(
+                        "Sent immediate ResendBlock response for block {} on simulator {}, last verified block: {}",
+                        blockNumber,
+                        nodeIndex,
+                        verifiedBlock);
+                break;
+            case SET_END_OF_STREAM_RESPONSE:
+                controller.setEndOfStreamResponse(nodeIndex, responseCode, blockNumber);
+                verifiedBlock = controller.getLastVerifiedBlockNumber(nodeIndex);
+                log.info(
+                        "Set EndOfStream response code {} for block {} on simulator {}, last verified block: {}",
+                        responseCode,
+                        blockNumber,
+                        nodeIndex,
+                        verifiedBlock);
+                break;
+            case RESET_RESPONSES:
+                controller.resetResponses(nodeIndex);
+                verifiedBlock = controller.getLastVerifiedBlockNumber(nodeIndex);
+                log.info("Reset all responses on simulator {} to default behavior", nodeIndex);
+                break;
+            case SHUTDOWN_SIMULATOR:
+                controller.shutdownSimulator(nodeIndex);
+                log.info("Shutdown simulator {} to simulate connection drop", nodeIndex);
```

Comment:
In this context we don't need to specify why we are shutting down; that should come from the caller. 

```suggestion
                log.info("Shutdown simulator {} ", nodeIndex);
```